module/VuFind/src/VuFind/Db/Row/User.php

```diff
@@ -784,6 +784,18 @@ public function getLoginTokens(string $userId): array
         return $tokenTable->getByUserId($userId);
     }
 
+    /**
+     * Get a user attribute by field name.
+     *
+     * @param string $fieldName Field name
+     *
+     * @return mixed
+     */
+    public function getField($fieldName)
```

Comment:
Similarly, could we avoid adding getField() by just adding a switch to pick the appropriate getter based on the provided field name in the calling code? A long-term goal of the entity classes is to avoid exposing internal details like raw field names, so I think it would be best not to set this precedent here if there's a reasonably simple alternative.